### PR TITLE
Fast-Integration-test refactoring after removal of SC: Helm-Broker

### PR DESCRIPTION
**Pull request by tillknuesting**
helm-Broker

### CODE REVIEW

File: tests/fast-integration/skr-svcat-migration-test/deploy-sample-resources.js [hunk 1]
```diff
@@ -252,7 +249,6 @@ async function deploy() {
   await waitForPodWithLabel('app', 'service-broker-proxy-k8s', 'kyma-system');
 
   try {
-    await waitForClusterServiceBroker('helm-broker', 5 * 60 * 1000);
```
Comment (wozniakjan) [inline tests/fast-integration/skr-svcat-migration-test/deploy-sample-resources.js:255]: nit: technically this whole test can be removed after svcat is removed from kyma but I think we can start iteratively and disable it by parts

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve

**Review by pbochynski — APPROVED** [review]

**Pull request merged**